aodndata/aatams/aatams_sattag_qc_dm_schema.py

```diff
@@ -375,19 +381,19 @@ def is_equal(x, y):
     "pass.dur": CSV_INT,
     "freq": CSV_POSITIVE_FLOAT,
     "v.mask": CSV_INT,
-    "alt": CSV_EMPTY,  # TODO
-    "est.speed": Or(CSV_EMPTY, CSV_FLOAT),  # TODO check range.
+    "alt": CSV_EMPTY,
+    "est.speed": Or(CSV_EMPTY, CSV_FLOAT),
     "km.from.home": Or(CSV_EMPTY, CSV_FLOAT),
     "iq": CSV_INT,
     "nops": CSV_INT,
-    "deleted": CSV_EMPTY,  # TODO
+    "deleted": CSV_EMPTY,
     "actual.ptt": CSV_INT,
     "error.radius": CSV_INT,
     "semi.major.axis": CSV_INT,
     "semi.minor.axis": CSV_INT,
     "ellipse.orientation": CSV_INT,
     "hdop": CSV_INT,
-    "satellite": str,  # TODO
```

Comment:
we're not harvesting these fields into our DB, should be empty or a string

Reply:
done